_includes/master/entityrule.md

```diff
@@ -7,3 +7,10 @@
 | namespaceSelector | Positive match on selected namespaces. If specified, only workload endpoints in the selected Kubernetes namespaces are matched. | Valid selector | [selector](#selector) | |
 | ports | Positive match on the specified ports | | list of [ports](#ports) | |
 | notPorts | Negative match on the specified ports | | list of [ports](#ports) | |
+
+##### Namespace Selector
```

Comment:
A bit odd having this be the only thing outside of the table - could we just enhance the description in the table instead?